### ISSUE TITLE
All retrieved events are marked as transparent

### ISSUE THREAD

**Issue by paulvt** [northworld/google_calendar#298]
It seems that the Google Calendar API only mentions in event JSON when the event is actually transparent. If the event is opaque, it is not part of the event JSON.
However, then for the opaque events `Event.new_from_feed` passes `nil` as value to `transparency=` (via `initialize`) which will then default to it setting `@transparency` to `"transparent"` which is the wrong outcome.  
As a result, all retrieved events are marked as transparent.

A solution would be to have `initialize` do:
```ruby
self.transparency = params[:transparency] || "opaque"
```

Another solution would be to have `new_from_feed` not parse it as part of the standard list of `params` but have an explicit line:
```ruby
params[:transparency] = e['transparency'] || "opaque"
```

I could not find in the docs whether it transparancy actually should default to "opaque" (the UI does). If this case, maybe event's `transparency=` could be adapted to use:
```ruby
if val.nil? || val == false || val.to_s.downcase == 'opaque'
```

**Comment by szich** [comment]
Good catch — this was fixed in commit 540fa13, which switched the transparency default to `opaque` per the [API reference](https://developers.google.com/calendar/v3/reference/events#transparency). Events built from a feed that omits the `transparency` field now resolve to `opaque` rather than `transparent`. Added feed-level regression tests in #303 to lock this in. Thanks for the detailed write-up!
